package/firmware/linux-firmware/intel.mk

```diff
@@ -202,7 +202,7 @@ $(eval $(call BuildPackage,iwlwifi-firmware-ax210))
 Package/iwlwifi-firmware-be200 = $(call Package/firmware-default,Intel BE200 firmware)
 define Package/iwlwifi-firmware-be200/install
 	$(INSTALL_DIR) $(1)/lib/firmware
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/iwlwifi-gl-c0-fm-c0-83.ucode $(1)/lib/firmware
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/iwlwifi-gl-c0-fm-c0-90.ucode $(1)/lib/firmware
```

Comment:
Please use version 92 instead. That is the highest version supported by the iwlwifi driver we are using.

Reply:
saw there is 92, but last month used/tested 90 - because of that push 90

Reply:
done